src/app.js

```diff
@@ -1 +1,29 @@
 'use strict';
+
+// instruction in how to use app
+// sourcePath <string> source filename to copy
+// destinationPath <string> destination filename of the copy operation
+// in command line type 'node src/app.js sourcePath destinationPath'
+// example: command 'node src/app.js src/dir1/file.txt src/dir2/file-copy.txt'
+// will copy file from dir1 to dir2
+
+const copyFile = require('fs-copy-file');
+
+const source = process.argv[2];
+const dest = process.argv[3];
+
+const callback = (err) => {
```

Comment:
```suggestion
const callback = (error) => {
```
2 cut chars will not save the situation)
Informativeness is a priority